### DISCUSSION TITLE
Why remove the -1 factor in legendre?

### DISCUSSION THREAD

**Discussion by liangzhixin-202169** [e3nn/e3nn#435] [Q&A]
The _sympy_legendre(l, m) code says that "remove two times (-1)^m" and "remove (-1)^l", the normalization P(l, m)  formula should  satisfy P(l, -m) = (-1)^m*P(l, m), so I wonder why they can be moved.

**Comment by mariogeiger** [comment]
I see, well I guess that `_sympy_legendre` is missing those terms but when we use the function we probably counter act.

It was just done like that to simplify the code, maybe not the best idea but I'm sure that the overall spherical harmonics (that use this function) are nevertheless correct.
